spec/requests/splash_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Splashes', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
- [ ] I can see that some tests are pending as a result of no test written for those files. I will encourage you to delete those files since you did not write tests for them. We should have our tests all green.

Reply:
Thanks !